locales/es.json

```diff
@@ -11,33 +11,33 @@
    "Welcome to Embark": "Bienvenido a Embark",
    "possible commands are:": "los comandos posibles son:",
    "display versions in use for libraries and tools like web3 and solc": "mostrar versiones en uso para bibliotecas y herramientas como web3 y solc",
-   "instantiated web3.js object configured to the current environment": "objeto web3.js instanciado configurado para el entorno actual",
+   "instantiated web3.js object configured to the current environment": "objeto web3.js instanciado ha sido configurado para el entorno actual",
    "to immediatly exit (alias: exit)": "para salir inmediatamente (alias: exit)",
-   "The web3 object and the interfaces for the deployed contracts and their methods are also available": "El objeto web3 y las interfaces para los contratos publicados y sus métodos también están disponibles",
+   "The web3 object and the interfaces for the deployed contracts and their methods are also available": "El objeto web3 y las interfaces para los contratos desplegados y sus métodos también están disponibles",
    "create a working dapp with a SimpleStorage contract": "Crear una dapp funcional con un contrato SimpleStorage",
-   "filename to output logs (default: none)": "fichero/archivo para salida de los logs (predefinido: none)",
+   "filename to output logs (default: none)": "archivo de escritura de logs (Por defecto: none)",
```

Comment:
-Line 18 should be: “nombre de archivo de los registros de salida (por defecto: ninguno)”